scripts/lockfile-generators/download-pip-packages.py

```
@@ -0,0 +1,208 @@
+    subprocess.run(["wget", "-q", "-O", str(path), url], check=True)
+
+
+def main():
+    req_path, out_dir = get_and_validate_args()
+
+    # Detect --index-url in requirements file (e.g. RHOAI)
+    index_url = detect_index_url(req_path)
+    use_simple_index = index_url is not None and "pypi.org" not in index_url
+    if use_simple_index:
+        print(f"Detected custom index: {index_url}")
+        print(f"Using PEP 503 simple index for downloads.\n")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)